cluster/src/main/scala/io/snappydata/remote/interpreter/SnappyInterpreterExecute.scala

```diff
@@ -141,6 +142,25 @@ object SnappyInterpreterExecute {
     } finally {
       if (lockTaken) intpRWLock.writeLock().unlock()
     }
+    updateMetaRegion(group)
+  }
+
+  private def updateMetaRegion(group: String): Unit = {
+    val r = Misc.getMemStore.getMetadataCmdRgn
+    val allAuthKeys = r.keySet().asScala.filter(s =>
+      s.startsWith(GrantRevokeOnExternalTable.META_REGION_KEY_PREFIX))
+    allAuthKeys.foreach(k => {
+      val p = r.get(k)
+      if (p != null) {
+        p.asInstanceOf[PermissionChecker].addLdapGroup(getNameWithLDAPPrefix(group), true)
```

Comment:
As an optimization, LDAP groups members iterator can be obtained only once and used everywhere while updating the list (instaed of getting it once in "permissions.refreshOnLdapGroupRefresh(group)" and again in "SnappyInterpreterExecute.updateMetaRegion" via call to "addLdapGroup". 
Can be done later.

Reply:
Yes, will take care of it post 1.2.0.